nservicebus/bridge/performance.md

```diff
@@ -1,48 +1,47 @@
 
-The bridge accesses messages using NServiceBus transport packages. Each transport on its own will try to ensure that only one instance will actually process the message.
+The bridge accesses messages using NServiceBus transport packages, which ensure that only one instance processes a given message.
 
 NOTE: MSMQ is not a brokered transport and cannot benefit from the competing consumer pattern.
 
 ### Splitting up the bridge
 
-It is possible to use different logical instances of the bridge. Each logical instance can be configured so that it mimics only certain endpoints. In the diagram below, one bridge only takes care of Endpoint B and Endpoint C, while the other takes care of Endpoint A and Endpoint D. The result (which is not visible in the diagram) is that Endpoint B can still send messages to Endpoint D, but those will be processed by Bridge 2. When Endpoint A sends a message to Endpoint C, those messages will be processed by Bridge 1.
-
-This can be useful when resources such as a CPU or disk are fully utilized and the competing consumers pattern isn't the appropriate solution.
-
-NOTE: It may be practical to split up logical endpoints to increase throughput for *specific messages*. For the bridge this benefit is lost, as each endpoint that is mimicked by the bridge has its own message processing pipeline, separate from other mimicked endpoints.
+It is possible to use multiple logical instances of the bridge. Each logical instance may be configured to mimic only specific endpoints. In the diagram below, one bridge mimics only endpoints B and C, and the other mimics endpoints A and D. The result (which is not visible in the diagram) is that endpoint B can still send messages to endpoint D, but those are processed by bridge 2. When endpoint A sends a message to endpoint C, those messages are processed by bridge 1.
 
+This can be useful when resources such as a CPU and disk are fully utilized and the competing consumers pattern isn't an appropriate solution.
 
+NOTE: It may be useful to split up logical endpoints to increase throughput for *specific messages*. For the bridge, this benefit is lost, as each endpoint that is mimicked by the bridge has its own message processing pipeline, separate from other mimicked endpoints.
 
 ```mermaid
 graph LR
-  Br2[Bridge 2<br /><sup>Endpoint A<br />Endpoint D</sup>]
+  Br2[Bridge 2]
   A[Endpoint A] --- Br2
   B[Endpoint B]
   Br2 --- D[Endpoint D]
-  Br1[Bridge 1<br /><sup>Endpoint B<br />Endpoint C</sup>] --- C[Endpoint C]
+  Br1[Bridge 1] --- C[Endpoint C]
   B --- Br1
```

Comment:
## Before

![image](https://user-images.githubusercontent.com/677704/193579808-8db5699c-98a9-4149-a01f-eb5522011a4b.png)

The extra annotations in the bridge boxes really don't add anything, and they're quite ugly.

## After

![image](https://user-images.githubusercontent.com/677704/193579676-3de3d79f-9d51-4347-b969-1d7c91dc6484.png)